docs/index.md

```diff
@@ -1,241 +1,172 @@
-data; it's about having the *right* data, presented in ways that make sense for Python applications.
+To install the latest version of the **Logfire** SDK, run:
 
-### In the Spirit of Python
+{{ install_logfire() }}
 
-- [**Simplicity and Power**](#simplicity): Emulating the Pydantic library's philosophy, Pydantic Logfire offers an
-intuitive start for beginners while providing the depth experts desire. It's the same balance of ease, sophistication,
-and productivity, reimagined for observability.
-- **Born from Python and Pydantic**: As creators immersed in the Python open-source ecosystem, we've designed Pydantic
-Logfire to deeply integrate with Python and Pydantic's nuances, delivering a more customized experience than generic
-observability platforms.
+## Development / Production
+=== "Development"
+    !!! tip "Development setup"
+        During development, we recommend using the CLI to configure Logfire. You can also use a write token.
 
-### Elevating Data to Insights
+    ## Authenticate
 
-- With [**deep Python integration**](#python), Pydantic Logfire automatically instruments your code for minimal manual
-effort, provides exceptional insights into async code, offers detailed performance analytics, and displays Python
-objects the same as the interpreter. For existing Pydantic users, it also delivers unparalleled insights into your usage
-of Pydantic models.
-- [**Structured Data and Direct SQL Access**](#sql) means you can use familiar tools like Pandas, SQLAlchemy, or `psql`
-for querying, can integrate seamlessly with BI tools, and can even leverage AI for SQL generation, ensuring your Python
-objects and structured data are query-ready. Using vanilla PostgreSQL as the querying language throughout the platform
-ensures a consistent, powerful, and flexible querying experience.
-- By [**harnessing OpenTelemetry**](#otel), Pydantic Logfire offers automatic instrumentation for popular Python packages,
-enables cross-language data integration, and supports data export to any OpenTelemetry-compatible backend or proxy.
+    Authenticate your local environment with **Logfire** by running:
 
-## Pydantic Logfire: The Observability Platform You Deserve
+    ```bash
+    logfire auth
+    ```
 
-**Pydantic Logfire** isn't just another tool in the shed; it's the bespoke solution crafted by Python developers, for
-Python developers, ensuring your development work is as smooth and efficient as Python itself.
+    !!! info
+        Upon successful authentication, credentials are stored in `~/.logfire/default.toml`.
 
-From the smallest script to large-scale deployments, Pydantic Logfire is the observability solution you've been waiting
-for.
+    ## Configure your project
 
----
+    Point to the project you created by running:
 
-### Simplicity and Power :rocket: {#simplicity}
+    ```bash
+    logfire projects use first-project
+    ```
```

Comment:
```suggestion
    ```

    INFO: this will store a Write Token which the SDK can use to end data in a file in the current directory at `.logfire/logfire_credentials.json`
```

Reply:
I added this under an info call-out in a separate commit.